src/ops/add.js

```diff
@@ -4,14 +4,16 @@ const path = require('path')
 const fs = require('fs-extra')
 const { red } = require('chalk')
 const add = async (action: RenderedAction, args, { logger, cwd }) => {
-  const { attributes: { to, inject } } = action
+  const { attributes: { to, inject, unless_exists } } = action
   if (!to || inject) {
     return
   }
   const absTo = path.join(cwd, to)
+  const shouldNotOverwrite = (typeof unless_exists !== 'undefined') && unless_exists === true
```

Comment:
I'm wondering why not this (am I missing something?):

```
unless_exists !== undefined
```

Reply:
Shouldn't be an issue in this case probably. But I normally do it that way to prevent errors from beeing triggered.

Reply:
Yup only if it's quick and you don't mind, please change it. Only because my first reaction was "did I miss a best practice that I didn't get a chance to learn?" makes it a bit unsurprising.